src/components/message/Message.tsx

```diff
@@ -23,18 +23,24 @@ interface MessageProps {
   sx?: {
     avatar?: SxProps
   }
+  filteredMessages: string[]
```

Comment:
Add it as not important because we will see warning

Reply:
fixed